### ISSUE TITLE
attribute-deprecated False positive

### ISSUE THREAD

**Issue by voronind** [OCA/pylint-odoo#339]
This code raises `attribute-deprecated`

```python
# Just module scope function
def func(a):
    length = len(a)
    return length
```

**Comment by voronind** [comment]
:+1:
